### ISSUE TITLE
[noname-clubl] (testing) Exception (noname-clubl): Unsupported POST Content-Type: application/x-www-form-urlencoded: Unsupported POST Content-Type: application/x-www-form-urlencoded

### ISSUE THREAD

**Issue by Anton111111** [Jackett/Jackett#13304]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Starting Jackett v0.20.1197
Environment version: 6.0.6 (/app/Jackett/)
OS version: Unix 5.4.17.2102 (64bit OS) (64bit process)
Jackett variant: CoreLinuxArm64
File /etc/issue: Ubuntu 20.04.4 LTS \n \l
Running in Docker: Yes (image build: v0.20.1197-ls70)
ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
App config/log directory: /config/Jackett
Using proxy: Disabled
Using FlareSolverr: http://flaresolverr:8191/
Using HTTP Client: HttpWebClient2
```

### Description

When i try test noname-clubl i get error:
```
An error occurred while testing this indexer
Exception (noname-clubl): Unsupported POST Content-Type: application/x-www-form-urlencoded: Unsupported POST Content-Type: application/x-www-form-urlencoded
```

### Logged Error Messages

```text
Jackett.Common.IndexerException: Exception (noname-clubl): Unsupported POST Content-Type: application/x-www-form-urlencoded
 ---> FlareSolverrSharp.Exceptions.FlareSolverrException: Unsupported POST Content-Type: application/x-www-form-urlencoded
   at FlareSolverrSharp.Solvers.FlareSolverr.GenerateFlareSolverrRequest(HttpRequestMessage request, String sessionId)
   at FlareSolverrSharp.Solvers.FlareSolverr.Solve(HttpRequestMessage request, String sessionId)
   at FlareSolverrSharp.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
   at Jackett.Common.Utils.Clients.HttpWebClient2.Run(WebRequest webRequest) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs:line 178
   at Jackett.Common.Utils.Clients.WebClient.GetResultAsync(WebRequest request) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/WebClient.cs:line 209
   at Jackett.Common.Indexers.BaseWebIndexer.RequestWithCookiesAsync(String url, String cookieOverride, RequestType method, String referer, IEnumerable`1 data, Dictionary`2 headers, String rawbody, Nullable`1 emulateBrowser) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 516
   at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1399
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query, Boolean isMetaIndexer) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 310
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query, Boolean isMetaIndexer) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 322
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query, Boolean isMetaIndexer) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 677
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/vsts/work/1/s/src/Jackett.Common/Services/IndexerManagerService.cs:line 308
   at Jackett.Server.Controllers.IndexerApiController.Test() in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 132
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 61
```

**Comment by ilike2burnthing** [comment]
Reconfigure the indexer and try again.

**Comment by LuanHimmlisch** [comment]
Having the same issue

**Comment by ilike2burnthing** [comment]
@LuanHimmlisch see above comment

**Comment by LuanHimmlisch** [comment]
@ilike2burnthing that doesn't work

**Comment by ilike2burnthing** [comment]
Working ok here:
- test - fails
- click Configure button - FlareSolverr detects and solves challenge
- re-enter login details and click Okay - successful
- test - successful

Which of these steps is failing for you?

**Comment by Anton111111** [comment]
@ilike2burnthing , I removed indexer. But now i tried to add it again. My Steps:
1) click Configure button
2) Click on Okey
And i got error: FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)

But in logs  i see that challenge solved.

Full logs:
```
flaresolverr  | 2022-06-18T06:16:01+00:00 INFO REQ-9 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://nnmclub.to/forum/login.php"}
flaresolverr  | 2022-06-18T06:16:07+00:00 INFO REQ-9 Cloudflare detected
flaresolverr  | 2022-06-18T06:16:46+00:00 INFO REQ-9 Challenge solved
flaresolverr  | 2022-06-18T06:17:02+00:00 WARN REQ-9 Page not reloaded (do not report!): Cause: Error: Protocol error (Page.navigate): Session closed. Most likely the page has been closed.
flaresolverr  | 2022-06-18T06:17:02+00:00 ERROR REQ-9 Unexpected error: Error: Protocol error (Network.getCookies): Session closed. Most likely the page has been closed.
flaresolverr  | 2022-06-18T06:17:03+00:00 ERROR REQ-9 Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
flaresolverr  | 2022-06-18T06:17:03+00:00 INFO REQ-9 Response in 61.797 s
jackett       | 06-18 06:17:03 Error FlareSolverrSharp.Exceptions.FlareSolverrException: FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
jackett       |    at FlareSolverrSharp.Solvers.FlareSolverr.<>c__DisplayClass10_0.<<SendFlareSolverrRequest>b__0>d.MoveNext()
jackett       | --- End of stack trace from previous location ---
jackett       |    at FlareSolverrSharp.Utilities.SemaphoreLocker.LockAsync[T](Func`1 worker)
jackett       |    at FlareSolverrSharp.Solvers.FlareSolverr.SendFlareSolverrRequest(HttpContent flareSolverrRequest)
jackett       |    at FlareSolverrSharp.Solvers.FlareSolverr.Solve(HttpRequestMessage request, String sessionId)
jackett       |    at FlareSolverrSharp.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
jackett       |    at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
jackett       |    at Jackett.Common.Utils.Clients.HttpWebClient2.Run(WebRequest webRequest) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs:line 178
jackett       |    at Jackett.Common.Utils.Clients.WebClient.GetResultAsync(WebRequest request) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/WebClient.cs:line 209
jackett       |    at Jackett.Common.Indexers.BaseWebIndexer.RequestWithCookiesAsync(String url, String cookieOverride, RequestType method, String referer, IEnumerable`1 data, Dictionary`2 headers, String rawbody, Nullable`1 emulateBrowser) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 516
jackett       |    at Jackett.Common.Indexers.CardigannIndexer.GetConfigurationForSetup(Boolean automaticlogin) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 905
jackett       |    at Jackett.Common.Indexers.CardigannIndexer.DoLogin() in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 576
jackett       |    at Jackett.Common.Indexers.CardigannIndexer.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 981
jackett       |    at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 97
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
jackett       |    at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
jackett       |    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
jackett       |    at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 61 FlareSolverrSharp.Exceptions.FlareSolverrException: FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Unable to process browser request. Error: Maximum timeout reached. maxTimeout=55000 (ms)
jackett       |    at FlareSolverrSharp.Solvers.FlareSolverr.<>c__DisplayClass10_0.<<SendFlareSolverrRequest>b__0>d.MoveNext()
jackett       | --- End of stack trace from previous location ---
jackett       |    at FlareSolverrSharp.Utilities.SemaphoreLocker.LockAsync[T](Func`1 worker)
jackett       |    at FlareSolverrSharp.Solvers.FlareSolverr.SendFlareSolverrRequest(HttpContent flareSolverrRequest)
jackett       |    at FlareSolverrSharp.Solvers.FlareSolverr.Solve(HttpRequestMessage request, String sessionId)
jackett       |    at FlareSolverrSharp.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
jackett       |    at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
jackett       |    at Jackett.Common.Utils.Clients.HttpWebClient2.Run(WebRequest webRequest) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs:line 178
jackett       |    at Jackett.Common.Utils.Clients.WebClient.GetResultAsync(WebRequest request) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/WebClient.cs:line 209
jackett       |    at Jackett.Common.Indexers.BaseWebIndexer.RequestWithCookiesAsync(String url, String cookieOverride, RequestType method, String referer, IEnumerable`1 data, Dictionary`2 headers, String rawbody, Nullable`1 emulateBrowser) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 516
jackett       |    at Jackett.Common.Indexers.CardigannIndexer.GetConfigurationForSetup(Boolean automaticlogin) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 905
jackett       |    at Jackett.Common.Indexers.CardigannIndexer.DoLogin() in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 576
jackett       |    at Jackett.Common.Indexers.CardigannIndexer.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 981
jackett       |    at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 97
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
jackett       |    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
jackett       |    at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
jackett       |    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
jackett       |    at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 61

```

**Comment by ilike2burnthing** [comment]
Are you able to log into the site in browser from the same device and IP as Jackett is installed on, using the same DNS?

Add the public indexer `Badass Torrents` and perform a keyword search with it, e.g. `720p`. Is FlareSolverr successful?

**Comment by Anton111111** [comment]
@ilike2burnthing , I can't test in browser because jackett started in cloud instance.  

Search and FlareSolverr works good with Badass. Here is logs:
```
flaresolverr  | 2022-06-20T05:32:47+00:00 INFO REQ-11 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://badasstorrents.com/torrents/search/720p/date/desc"}
flaresolverr  | 2022-06-20T05:32:57+00:00 INFO REQ-11 Cloudflare detected
flaresolverr  | 2022-06-20T05:33:04+00:00 INFO REQ-11 Challenge solved
flaresolverr  | 2022-06-20T05:33:10+00:00 INFO REQ-11 Response in 22.929 s
jackett       | 06-20 05:33:29 Info Manual search in Badass Torrents for 720p => Found 100 releases
jackett       | 06-20 05:34:05 Info Manual search in Badass Torrents for 1080p => Found 100 releases
```

**Comment by ilike2burnthing** [comment]
Is this running on a VPS?

FlareSolverr is working, and you're not being blocked by Cloudflare from accessing the site, it's just that once the challenge is resolved, reloading the page is timing out, and I'm not sure why.

**Comment by Anton111111** [comment]
@ilike2burnthing , Yes it is VPS.

Now i just removed noname-clubl.

**Comment by ilike2burnthing** [comment]
Out of interest, does the public noname-club work for you?

**Comment by Anton111111** [comment]
@ilike2burnthing , public noname-club show same error like private.
```
Exception (noname-club): Unsupported POST Content-Type: application/x-www-form-urlencoded: Unsupported POST Content-Type: application/x-www-form-urlencoded
```

**Comment by ilike2burnthing** [comment]
Not currently, and not any time soon - https://github.com/Jackett/Jackett/issues/2539

**Comment by LinFor** [comment]
I have the same issue with noname-club and noname-clubl indexers. badasstorrents working fine, flaresolverr working fine. Proxy not used.
nnmclub.to opens fine in browser from the same public ip.

**Comment by LinFor** [comment]
> Maybe it needs to re-solve cloudflare challenge but it somehow bypasses flaresolverr and tries to perform request directly? Re-adding noname-clubl (and re-solving cf challenge using flaresolverr) fixes issue for some time for me. (arch linux, native jackett, flaresolverr, all from aur).

I noticed that setting the address via http (without S) in the settings helps. It resets itself to https, but the search works for a while.
I think it's a problem with https and flaresolverr.

**Comment by NoAdO** [comment]
Had a same issue in docker installation, tryed to re-add indexer.
It takes a long time to show configuration windows - about a minute or more.
When i click "ok" Jacket thinks some time and then shows error
`An error occurred while updating this indexer
FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Cloudflare Error: No challenge selectors found, unable to proceed.`

Flresolver logs this:
`2022-07-30T13:08:32+03:00 INFO REQ-6 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://nnmclub.to/forum/login.php"}
2022-07-30T13:08:43+03:00 INFO REQ-6 Cloudflare detected
2022-07-30T13:09:58+03:00 INFO REQ-6 Cloudflare Error: No challenge selectors found, unable to proceed.
2022-07-30T13:09:58+03:00 INFO REQ-6 Response in 85.91 s
2022-07-30T13:10:38+03:00 INFO REQ-7 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://nnmclub.to/forum/login.php"}
2022-07-30T13:10:50+03:00 INFO REQ-7 Cloudflare detected
2022-07-30T13:12:03+03:00 INFO REQ-7 Cloudflare Error: No challenge selectors found, unable to proceed.
2022-07-30T13:12:03+03:00 INFO REQ-7 Response in 84.989 s`

I can reach nnmclub via browser
There's no SOCKS configured
Adding nnm with http instead of https changes nothing
badasstorrents added instantly with no errors but search of "720" returned 0 results. Same does public nnm. Other trackers (ex. rutracker, rutor) keep working and give results.

Is this a Jacket of FlareSolverr problem?

**Comment by ilike2burnthing** [comment]
@ngosang using FlareSolverr 2.2.5 and Jackett v0.20.1496 trying to add or reconfigure this indexer now results in:

```text
System.UriFormatException: Invalid URI: The hostname could not be parsed.
   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind, UriCreationOptions& creationOptions)
   at System.Uri..ctor(String uriString)
   at Jackett.Common.Utils.CookieUtil.RemoveAllCookies(CookieContainer cookieJar) in D:\a\1\s\src\Jackett.Common\Utils\CookieUtil.cs:line 56
   at Jackett.Common.Utils.Clients.HttpWebClient2.Run(WebRequest webRequest) in D:\a\1\s\src\Jackett.Common\Utils\Clients\HttpWebClient2.cs:line 119
   at Jackett.Common.Utils.Clients.WebClient.GetResultAsync(WebRequest request) in D:\a\1\s\src\Jackett.Common\Utils\Clients\WebClient.cs:line 186
   at Jackett.Common.Indexers.BaseWebIndexer.RequestWithCookiesAsync(String url, String cookieOverride, RequestType method, String referer, IEnumerable`1 data, Dictionary`2 headers, String rawbody, Nullable`1 emulateBrowser) in D:\a\1\s\src\Jackett.Common\Indexers\BaseIndexer.cs:line 512
   at Jackett.Common.Indexers.CardigannIndexer.GetConfigurationForSetup(Boolean automaticlogin) in D:\a\1\s\src\Jackett.Common\Indexers\CardigannIndexer.cs:line 910
   at Jackett.Common.Indexers.CardigannIndexer.DoLogin() in D:\a\1\s\src\Jackett.Common\Indexers\CardigannIndexer.cs:line 581
   at Jackett.Common.Indexers.CardigannIndexer.ApplyConfiguration(JToken configJson) in D:\a\1\s\src\Jackett.Common\Indexers\CardigannIndexer.cs:line 986
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in D:\a\1\s\src\Jackett.Server\Controllers\IndexerApiController.cs:line 97
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in D:\a\1\s\src\Jackett.Server\Middleware\CustomExceptionHandler.cs:line 61
```

**Comment by ngosang** [comment]
@ilike2burnthing I'm on it

**Comment by ngosang** [comment]
@ilike2burnthing could yo provide me the URL you are configuring in the indexer?

**Comment by ilike2burnthing** [comment]
https://nnmclub.to/ with error occurring for https://nnmclub.to/forum/login.php

**Comment by ngosang** [comment]
@ilike2burnthing I can't reproduce the issue. Could you test this PR and give me back the Warning traces?
https://github.com/Jackett/Jackett/pull/13420

**Comment by Nik-mmzd** [comment]
@ngosang 
> I can't reproduce the issue.

You need to wait for some time to reproduce the issue:
1. Add nnm-clubl
2. Test: all works
3. Wait for some time (I discovered the issue after some hours)
4. Try to search anytning with previously added nnm-clubl
5. You'll got the error and if you'll re-add nnm-clubl... goto 1

PS did not tested #13420 (but will give it a try), just describing the issue

**Comment by ilike2burnthing** [comment]
Fixed by #13425.

@Anton111111 @Nik-mmzd please add or reconfigure the indexer with [v0.20.1504](https://github.com/Jackett/Jackett/releases/tag/v0.20.1504) and test it's now working.

**Comment by ilike2burnthing** [comment]
@ngosang `Unsupported POST Content-Type: application/x-www-form-urlencoded` issue resolved for the semi-private noname-clubl but not for the public noname-club.

**Comment by NoAdO** [comment]
Trying to add nnmclub after updating both docker containers. Can see flaresolverr log in 5 seconds after pushing "ok" button.
Jackett:
```Exception: System.Threading.Tasks.TaskCanceledException: The request was canceled due to the configured HttpClient.Timeout of 55 seconds elapsing. ---> System.TimeoutException: The operation was canceled. ```
```2022-07-31T12:40:54+03:00 INFO REQ-4 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://nnmclub.to/forum/login.php"}
2022-07-31T12:41:35+03:00 INFO REQ-4 Cloudflare detected
2022-07-31T12:41:58+03:00 INFO REQ-4 Challenge solved
2022-07-31T12:41:59+03:00 INFO REQ-4 Response in 65.326 s```
Challenge is solver but Jackett already timeouts. Well, i try a 120000 timeout instead of 55000. Next try is **successfull**
```2022-07-31T12:46:15+03:00 INFO REQ-5 Response in 82.759 s
2022-07-31T12:46:11+03:00 INFO REQ-5 Challenge solved
2022-07-31T12:45:37+03:00 INFO REQ-5 Cloudflare detected
2022-07-31T12:44:52+03:00 INFO REQ-5 Incoming request => POST /v1 body: {"maxTimeout":120000,"cmd":"request.get","url":"https://nnmclub.to/forum/login.php"}```
Indexer added. But i don't know why FlareSolverr need to much time to proceed request. Mb it's about my network specific, i'm using VPN to route traffic.

**Comment by ngosang** [comment]
Fixed in #13435 + FlareSolverr 2.2.6

**Comment by ilike2burnthing** [comment]
[v0.20.1518](https://github.com/Jackett/Jackett/releases/v0.20.1518)
